examples/jsm/loaders/SVGLoader.js

```diff
@@ -2436,4 +2447,132 @@ SVGLoader.pointsToStrokeWithBuffers = function () {
 
 }();
 
+SVGLoader.extrudeVertices = function ( extrudeOptions, vertices, uvs ) {
```

Comment:
Does need to be a static method of `SVGLoader`?

Reply:
Not really, this is just a helper for extruding planar vertices, perhaps it can be moved to a util file, or we can not export it.